### PR TITLE
Create 'all in one' docker image OPFIDO-178

### PR DESCRIPTION
**Pull request by dsummersl**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    docker run --rm \
+      -v /tmp:/tmp \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -p 127.0.0.1:5001:5001 \
+      -p 127.0.0.1:5002:5002 \
+      -p 127.0.0.1:5003:5003 \
+      -p 127.0.0.1:9000:9000 \
+      -p 127.0.0.1:3000:3000 \
+      zne17nct/openfido
```
Comment (krohling) [inline docker/README.md:17]: @dsummersl I think you need to update this to 'openfido/openfido'

Comment (krohling) [inline docker/README.md:17]:
```suggestion
      openfido/openfido
```

**Review by krohling — APPROVED** [review]

File: docker/README.md (hunk 1 above)
Comment (dsummersl) [reply docker/README.md:17]: I caught this in the seed pr, thankfully!

Comment (dsummersl) [reply docker/README.md:17]: but not here :( I'll update it.

**Pull request merged**